tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ElasticSearchViewsSpec.scala

```diff
@@ -222,6 +243,15 @@ class ElasticSearchViewsSpec extends BaseSpec with EitherValuable with CirceEq {
       }
     }
 
+    "the person resource created with no id in payload should have the default id in _source" in eventually {
+      deltaClient.post[Json](s"/views/$fullId2/test-resource:people/_search", matchAll, ScoobyDoo) { (json, response) =>
+        response.status shouldEqual StatusCodes.OK
+        val id       = hits(0)._id.string.getOption(json)
+        val sourceId = hits(0)._source.`@id`.string.getOption(json)
+        sourceId shouldEqual id
+      }
+    }
```

Comment:
It is not a bad idea for most of the tests but here it may slow down the test execution:
* When we create a view, it takes some time for the indexing routine to be started
* When the routine starts, it processes the other elements and batches things
* Elasticsearch itself has a refresh period to show the results

So do the setup before is in the end not a bad idea :)